homeassistant/components/alarm_control_panel/group.py

```diff
@@ -23,13 +26,15 @@ def async_describe_on_off_states(
 ) -> None:
     """Describe group on off states."""
     registry.on_off_states(
-        {
+        DOMAIN,
+        (
```

Comment:
Why change to a tuple and not a set? I think a set gives is a bit more flexibility if we want to change something later to avoid transformations to a set